src/attributes/codegen.md

```diff
@@ -352,3 +352,18 @@ trait object whose methods are attributed.
 [`core::intrinsics::caller_location`]: ../../core/intrinsics/fn.caller_location.html
 [`core::panic::Location::caller`]: ../../core/panic/struct.Location.html#method.caller
 [`Location`]: ../../core/panic/struct.Location.html
+
+## The `instruction_set` attribute
+
+The `instruction_set` attribute may be applied to a function to enable code generation for a specific
+instruction set supported by the target architecture. Currently, this is only available for `ARMv4T`
```

Comment:
Can you also include the syntax for the attribute? Something along the lines of "It uses the [MetaListPaths](attributes.md#meta-item-attribute-syntax) syntax to …" similar to how other attributes are defined.  

I would also drop the word "Currently" (everything in the reference is defining what is currently implemented).